src/core/blockchain/solana/client.ts

```diff
@@ -68,14 +69,20 @@ export class Client extends BlockchainGenericClient {
     }
 
     public async sendTransaction(transaction): Promise<{ txHash: string; rawResponse: any }> {
-        return this.connection.sendRawTransaction(transaction).then(res => {
-            if (res) {
-                return {
-                    txHash: res,
-                    rawResponse: res
-                };
-            }
-        });
+        return this.connection
+            .sendRawTransaction(transaction)
+            .then(res => {
+                if (res) {
+                    return {
+                        txHash: res,
+                        rawResponse: res
+                    };
+                }
+            })
+            .catch((error: any) => {
+                SentryCaptureException(new Error(JSON.stringify(error)));
```

Comment:
It would be nice to have in sentry error logs from all transaction broadcasts from all blockchains. So I would move this in util-actions.ts in `signAndSendTransactions()` on the `catch` clause of `sendTransaction`

Reply:
It seems that we already handle errors and send them to sentry. The problem actually was that in client we don't handle errors, we don't throw them. I have fix this. Check again